lib/fs.js

```diff
@@ -1013,29 +1013,47 @@ function stat(path, options = { bigint: false }, callback) {
   binding.stat(pathModule.toNamespacedPath(path), options.bigint, req);
 }
 
-function fstatSync(fd, options = { bigint: false }) {
+function fstatSync(fd, options = { bigint: false, throw: true }) {
   validateInt32(fd, 'fd', 0);
   const ctx = { fd };
   const stats = binding.fstat(fd, options.bigint, undefined, ctx);
-  handleErrorFromBinding(ctx);
+  if (options.throw === false) {
```

Comment:
Sorry for misunderstanding but could you use `throws` instead of `throw` to avoid conflicting with keyword?

Reply:
My bad. 😅